### DISCUSSION TITLE
Mini guida su richieste FOIA

### DISCUSSION THREAD

**Discussion by aborruso** [ondata/italian-public-sector-pnrr-data-guide#14] [Aggiornamenti]
Quando i dati non sono pubblicati, si possono richiedere con un FOIA.

Era quindi utile aggiungere un riferimento anche al vademecum. Grazie ad Alberto Pampalone per averlo curato:
https://pnrr.datibenecomune.it/strumenti/foia.html

**Comment by girolamodaschio** [comment]
Ciao, 
Se io non sapessi esattamente di quale documento ho bisogno,posso fare una richiesta anche generica? 
tipo, le delibere relative alla gara xxyy. 
Grazie,
Girolamo

**Reply by giuragu** [reply]
Ciao @girolamodaschio,
l'esempio che proponi non è poi tanto generico, nel senso che si riferisce ad una specifica gara che indicheresti.
Comunque la risposta al tuo quesito ritengo possa stare nella [Delibera 1309/2016 di ANAC](https://www.anticorruzione.it/portal/rest/jcr/repository/collaboration/Digital%20Assets/anacdocs/Attivita/Atti/determinazioni/2016/1309/del.1309.2016.det.LNfoia.pdf) dove si legge che:

<<"l’istanza di accesso civico identifica i dati, le informazioni o i documenti richiesti”; pertanto **non è ammissibile una richiesta meramente esplorativa, volta semplicemente a “scoprire” di quali informazioni l’amministrazione dispone**. 
Le richieste, inoltre, **non devono essere generiche, ma consentire l’individuazione del dato, del documento o dell’informazione, con
riferimento, almeno, alla loro natura e al loro oggetto**>>

Spero questa risposta possa risolvere il tuo dubbio

**Reply by girolamodaschio** [reply]
Ciao @giuragu e grazie per la pronta risposta.
Il senso della domanda che ponevo è: non so esattamente quali documenti vengono predisposti per la singola gara.

Quindi la richiesta di accesso agli atti che farei è:
Il sottoscritto Nome Cognome, nato a, residente a, codice fiscale.
Richiede l'accesso a bandi di gara, delibere, inviti riferiti alle seguenti gare d'appalto:

Codice CIG | Codice CUP | Oggetto


è una procedura corretta o è troppo generico?
grazie,
Girolamo

**Reply by giuragu** [reply]
Non è generico. Ritengo inoltre tu possa procedere tramite accesso civico semplice poichè si tratta di documenti che andrebbero pubblicati in amministrazione trasparente. In questo caso potresti mantenere la domanda generica e chiedere la pubblicazione degli atti di gara di cui al codice cig, cup ecc...

**Reply by girolamodaschio** [reply]
Grazie mille, gentilissimo.
Girolamo

**Reply by aborruso** [reply]
@girolamodaschio, alla fine di questo "giro" di richieste, raccontiamola questa cosa, perché così può diventare una procedura da emulare.
